src/pages/registration/resource_type_tab/components/JournalField.tsx

```diff
@@ -157,7 +169,10 @@ export const JournalField = ({ confirmedContextType, unconfirmedContextType }: J
               {
                 hasMoreHits: !!journalOptionsQuery.data?.totalHits && journalOptionsQuery.data.totalHits > searchSize,
                 onShowMoreHits: () => setSearchSize(searchSize + defaultChannelSearchSize),
-                isLoadingMoreHits: journalOptionsQuery.isFetching && !journalOptionsQuery.isPending,
+                isLoadingMoreHits:
+                  journalOptionsQuery.isFetching &&
+                  journalOptionsQuery.data?.hits &&
+                  searchSize > journalOptionsQuery.data.hits.length,
```

Comment:
Kan forkortes til dette?

```suggestion
                  journalOptionsQuery.data?.hits.length < searchSize
```

Reply:
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/afc00470-3c2e-49b3-ae60-2552f601500e)

Men kan ev. forkortes til `searchSize > (journalOptionsQuery.data?.hits ?? []).length` om du syns det er enklere?

Reply:
gjør samme på linje 136, så gir mening å dra ut dette til en egen const med tomt array som fallback

Reply:
Ah, glemte at vi får warnings. Tenkte å missbruke `undefined < searchSize`. Men siden det da blir litt bloated uansett så har jeg ikke noen formening om hvilken som er bedre, så du får bestemme :)